theme/themes/pxt/elements/loader.overrides

```diff
@@ -15,11 +15,16 @@
   box-shadow: none;
   border-radius: 0px;
   background: transparent @loaderImage no-repeat center center;
-  background-size: 100%;
+  background-size: @loaderImageSize;
  -webkit-animation: @loaderAnimation @loaderSpeed infinite linear;
  animation: @loaderAnimation @loaderSpeed infinite linear;
 }
 
+.ui.loader.main.msft:after {
+  background: transparent @loaderMsftImage no-repeat center center;
+  background-size: 99%;
```

Comment:
does the 1% here make a difference / what's the change here handling?

Reply:
Ah okay makes sense, might be worth a quick comment above it so if we run into it again we know why it's special cased a bit~